<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="DataSourceManagerImpl" format="xml" multifile-model="true">
    <data-source source="LOCAL" name="datasource" uuid="4ff5bdd8-5fef-4471-bb1e-d28d3646b0fa">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:$PROJECT_DIR$/data.sqlite3</jdbc-url>
    </data-source>
    <data-source source="LOCAL" name="identifier.sqlite" uuid="a03552d7-4cbf-4511-8d44-8c45077af375">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:identifier.sqlite</jdbc-url>
    </data-source>
    <data-source source="LOCAL" name="Django default" uuid="a1d5f3da-bc03-4f77-9d67-3fbcdd904d23">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <imported>true</imported>
      <remarks>$PROJECT_DIR$/tweetme2/settings.py</remarks>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:E:\PROJECTS\TwitterDjango\data.sqlite3</jdbc-url>
    </data-source>
  </component>
</project>